airbyte-connector-builder/connector_builder/generated/models/extra_models.py

```diff
@@ -0,0 +1,8 @@
+# coding: utf-8
+
+from pydantic import BaseModel
+
+class TokenModel(BaseModel):
```

Comment:
I think in this scenario it doesn't make a difference, because we're not exporting python objects to external consumers of this package. If we were, like in the CDK, it matters a lot more because it decouples internal implementation details (e.g which file or directory a class is in) from how a user consumes the package. 

Happy to add it if you know of other reasons it should be there, but it seemed like an effort not worth maintaining

Reply:
Fair enough to leave as is!